framework/doc/content/source/controls/MOOSEFMU.md

```diff
@@ -0,0 +1,145 @@
+# MOOSE FMU Interface
+
+The `MOOSEFMU` defines the `Moose2FMU` base class which contains the boilerplate needed to wrap a MOOSE simulation as a Functional Mock-up Unit (FMU). Users only need to implement their own `__init__` and `do_step` methods when deriving from `Moose2FMU`.
+
+### Initialization Parameters
+
+`Moose2FMU` accepts a number of keyword arguments that configure how the
+underlying MOOSE simulation is launched and interacted with:
+
+- `flag`: Optional flag that is forwarded to the MOOSE input deck.
```

Comment:
how to pass multiple?
-flag a -flag b ? or -flag a b ?

Reply:
is the rules for spaces same as for CLI args?

Reply:
> is the rules for spaces same as for CLI args?

Yes, this is just pass CLI args to MOOSE control. I can add an example here to show it.  I keep all the examples in moose_fmu folder. I can refer that here